aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1634,12 +1634,14 @@ impl AptosVM {
                 return Err(invalid_signature!("Groth16 VK has not been set on-chain"));
             }
 
-            keyless_validation::validate_authenticators(
-                self.pvk.as_ref().unwrap(),
-                &keyless_authenticators,
-                self.features(),
-                resolver,
-            )?;
+            if !self.is_simulation {
```

Comment:
in short yes.  the signature has to be invalid anyway.  the simulator checks that the signature with respect to the authenticator is invalid.  however in keyless, you have a bunch of validation logic outside of the authenticator.  there no reason to use it because it's not supposed to pass and it makes simulation harder to work (have to use a legit proof and its needlessly computationally expensive)

for example the current response
```

[
  {
    version: '1216049668',
    hash: '0xc74f42f2280ddf3764afb7d0595a481f326c69c64884f984af2e23e1a0b1e23d',
    state_change_hash: '0x0000000000000000000000000000000000000000000000000000000000000000',
    event_root_hash: '0x0000000000000000000000000000000000000000000000000000000000000000',
    state_checkpoint_hash: null,
    gas_used: '0',
    success: false,
    vm_status: 'INVALID_SIGNATURE\n' +
      'Execution failed with message: The ephemeral keypair has expired',
    accumulator_root_hash: '0x0000000000000000000000000000000000000000000000000000000000000000',
    changes: [],
    sender: '0x2702d121810b0dd82da7fe72abc0972cb087cd8f37b55367a8fb0b2faead0763',
    sequence_number: '3',
    max_gas_amount: '200000',
    gas_unit_price: '100',
    expiration_timestamp_secs: '1716913770',
    payload: {
      function: '0x1::aptos_account::transfer_coins',
      type_arguments: [Array],
      arguments: [Array],
      type: 'entry_function_payload'
    },
    signature: {
      public_key: '0x1b68747470733a2f2f6163636f756e74732e676f6f676c652e636f6d20df280a94dd75e9e243f99287dc58768713a1aa7ed79f5a55b549bfafb606aa26',
      signature: '0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008096980000000000000000027b7d000000000000000000200000000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
      type: 'single_sender'
    },
    events: [],
    timestamp: '1716913750271770'
  }
]
```

Reply:
Let's follow-up with a comment here (you can more or less copy-paste your message above)